packages/front-end/services/datasources.ts

```diff
@@ -979,6 +1036,12 @@ export function getInitialDatasourceResources({
       userIdTypes: datasource.settings?.userIdTypes || [],
     });
 
+  if (datasource.type === "clickhouse")
+    return getClickHouseInitialDatasourceResources({
+      table: "test_enriched_events",
```

Comment:
I made this into an env var now `CLICKHOUSE_MAIN_TABLE` so we can easily swap it over to the real deal.

Reply:
I'm gonna swap it but, since this is the front end, this would mean we have to maintain this env var in both env which is annoying.